src/mkdocstrings/inventory.py

```diff
@@ -46,9 +46,12 @@ def format_sphinx(self) -> str:
             uri = uri[: -len(self.name)] + "$"
         return f"{self.name} {self.domain}:{self.role} {self.priority} {uri} {dispname}"
 
+    regex = re.compile(r"^(.+?)\s+(\S+):(\S+)\s+(-?\d+)\s+(\S+)\s+(.*)$")
```

Comment:
We should probably add "sphinx" somewhere in the name since it's only related to Sphinx.

Reply:
Hmm, actually this whole file is only related to Sphinx. (if it was intended to be generic, I'm not seeing the generic-ness :sweat_smile: )

As I see it, it'd actually be good to make this file dedicated to Sphinx

Reply:
True, it's only doing Sphinx stuff for now, but could be used to generated other kinds of inventories later. If that's not something we want, then yeah we could probably make it Sphinx explicit.